examples/sid_visualization/README.md

```diff
@@ -0,0 +1,135 @@
+# SID Visualization Example
+
+## Prerequisites
+
+To run the demo you will need the following:
+- Docker
```

Comment:
I dont see anything about specific versions of docker-compose. Is that somewhere else?

Reply:
5 lines down:
```
docker-compose (Tested with version 1.29)
```